test/metabase/util/product_feedback_test.clj

```diff
@@ -0,0 +1,34 @@
+(ns metabase.util.product-feedback-test
+  "Tests for product_feedback.clj"
+  (:require
+   [cheshire.core :as json]
+   [clj-http.client :as http]
+   [clj-http.fake :as http-fake]
+   [clojure.test :refer :all]
+   [metabase.util.product-feedback :as product-feedback :refer [product-feeback-url]]))
+
+(deftest send-product-feedback-test
+  (with-redefs [product-feeback-url "http://test.example.org/api/v1/crm/product-feedback"]
+    (testing "Should return an error if the request fails"
+      (binding [http/request (fn [& _]
+                               (throw (Exception. "network issues")))]
+        (is (= {:status            "failed"
+                :error_code        "connection-error",
+                :error_details     "network issues",
+                :feedback_endpoint "http://test.example.org/api/v1/crm/product-feedback"}
+               (#'product-feedback/send-product-feedback "some feedback" "some-source" "some-email@example.org")))))
+    (testing "Should forward the data to the cloud endpoint"
```

Comment:
With this test I wanted to make sure we're sending the correct data.
I tried to look for similar tests but I couldn't find any, any feedback is welcome.
Basically I wanted to check if I called http/post with the correct parameter, my solution was to throw an error in the handler if it received the wrong data, so that it returned `{"status":"success"}` only with the correct data